spec/rails-settings-cached/setting_spec.rb

```diff
@@ -98,42 +98,42 @@
   end
 
   describe 'Default values' do
-    it "can work with default value" do
+    it 'can work with default value' do
       Setting.defaults[:bar] = @bar
-      Setting.bar.should == @bar
+      expect(Setting.bar).to eq @bar
     end
 
-    it "can use default value, when the setting it cached with nil value" do
+    it 'can use default value, when the setting it cached with nil value' do
       Setting.has_cached_nil_key
-      Setting.defaults[:has_cached_nil_key] = "123"
-      Setting.has_cached_nil_key.should == "123"
+      Setting.defaults[:has_cached_nil_key] = '123'
+      expect(Setting.has_cached_nil_key).to eq '123'
     end
 
-    it "#save_default" do
+    it '#save_default' do
       Setting.test_save_default_key
-      Setting.save_default(:test_save_default_key, "321")
-      Setting.where(:var => "test_save_default_key").count.should == 1
-      Setting.test_save_default_key.should == "321"
-      Setting.save_default(:test_save_default_key, "3211")
-      Setting.test_save_default_key.should == "321"
+      Setting.save_default(:test_save_default_key, '321')
+      expect(Setting.where(var: 'test_save_default_key').count).to eq 1
+      expect(Setting.test_save_default_key).to eq '321'
+      Setting.save_default(:test_save_default_key, '3211')
+      expect(Setting.test_save_default_key).to eq '321'
     end
   end
 
-  describe "Implementation by embeds a Model" do
-    it "can set values" do
+  describe 'Implementation by embeds a Model' do
+    it 'can set values' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.